skimage/exposure/histogram_matching.py

```diff
@@ -19,7 +19,7 @@ def _match_cumulative_cdf(source, template):
     return interp_a_values[src_unique_indices].reshape(source.shape)
 
 
-def match_histograms(image, reference, multichannel=False):
+def match_histograms(image, reference, *, multichannel=False):
```

Comment:
This was released with 0.15, so it is an API change https://scikit-image.org/docs/stable/api/skimage.transform.html#match-histograms

Reply:
No, skimage.**exposure**.match_histograms is a brand new function! =)

Reply:
sounds good